packages/core/src/Link/Link.stories.js

```diff
@@ -1,28 +1,27 @@
 import React from 'react'
-import { action } from '@storybook/addon-actions'
-import { Cartesian } from '@compositor/kit'
 
 import { Link, Button } from '..'
 import ForwardRefDemo from '../../storybook/utils/ForwardRefsDemo'
 
-const variations = { link: 'link', fill: 'fill', outline: 'outline' }
-const colors = {
-  primary: 'primary',
-  secondary: 'secondary',
-  text: 'text',
-  success: 'success',
-  error: 'error',
-  warning: 'warning',
-  alert: 'alert',
-  caution: 'caution',
-  notify: 'notify',
-  pricePrimary: 'pricePrimary',
-  priceSecondary: 'priceSecondary',
-  promoPrimary: 'promoPrimary',
-  promoSecondary: 'promoSecondary',
-  border: 'border',
-  background: 'background',
-}
+// for Args
+// const variations = { link: 'link', fill: 'fill', outline: 'outline' }
+// const colors = {
+//   primary: 'primary',
+//   secondary: 'secondary',
+//   text: 'text',
+//   success: 'success',
+//   error: 'error',
+//   warning: 'warning',
+//   alert: 'alert',
+//   caution: 'caution',
+//   notify: 'notify',
+//   pricePrimary: 'pricePrimary',
+//   priceSecondary: 'priceSecondary',
+//   promoPrimary: 'promoPrimary',
+//   promoSecondary: 'promoSecondary',
+//   border: 'border',
+//   background: 'background',
+// }
```

Comment:
same

Reply:
same - will go away once we port these to Args, we can do them in the workshop